patches/ui-views-widget-desktop_aura-desktop_native_widget_aura.cc.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc b/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
+index 0ca3d45fc747c9d2e11acc2af4d95f85aacc70b3..e1635cb36adf14d9f255169ba0303a6759ed96e0 100644
+--- a/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
++++ b/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
+@@ -537,10 +537,10 @@ void DesktopNativeWidgetAura::UpdateWindowTransparency() {
+       desktop_window_tree_host_->ShouldWindowContentsBeTransparent();
+ 
+   auto* window_tree_host = desktop_window_tree_host_->AsWindowTreeHost();
+-  window_tree_host->compositor()->SetBackgroundColor(
```

Comment:
is this an upstream patch?

Reply:
cc: @cdesouza-chromium (though we may also need feedback from @simonhong on this one)